backup/borgmatic.yaml

```diff
@@ -0,0 +1,44 @@
+---
+location:
+  source_directories:
+    - ${FTP_DIR}
+  repositories:
+    - ${BORG_REPO}
+  one_file_system: true
+  exclude_caches: true
+  exclude_if_present: .nobackup
+
+storage:
+  compression: auto,zstd
+  ssh_command: ssh -o StrictHostKeyChecking=no -i sshkey
+  archive_name_format: 'gamecon-{now:%Y-%m-%d}'
+  encryption_passphrase: ${BORG_PASS}
+
+retention:
+  keep_daily: 7
+  keep_weekly: 4
+  keep_monthly: 12
+  keep_yearly: 2
+  prefix: 'gamecon-'
+
+consistency:
+  checks:
+    - repository
+    - archives
+  check_last: 3
+  prefix: 'gamecon-'
+
+hooks:
+  before_backup:
+    - >-
```

Comment:
Rozšiř mi prosím obzory. Co dělá `>-` ?

Praktické testy ukázaly, že to dovoluje napsat víceřádkový command. Ale nevím proč.

Reply:
Víceřadkový příkaz, kde odřádkování je nahrazeno za mezery (https://yaml.org/spec/1.2-old/spec.html#style/block/folded). Ideální na rozdělení jednoho příkazu na více řádků.

Reply:
A ta pomlčka?

Reply:
Ta pomlčka sežere koncový newline 😉